### PR TITLE
Fix examples imports

### PR DESCRIPTION
**Pull request by GijsGroote**
Names have changed, the examples did not work any more because of changed names. 

This is fixed in this PR.

### CODE REVIEW

File: examples/obstacle_sensor.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import gym
-import urdfenvs.pointRobotUrdf
+import urdfenvs.point_robot_urdf
 from urdfenvs.sensors.obstacleSensor import ObstacleSensor
```
Comment (maxspahn) [inline examples/obstacle_sensor.py:3]: It is simply the import that is wrong.
It should be `from urdfenvs.sensors.obstacle_sensor import ObstacleSensor`
Then you don't need the extra file in sensors.

File: examples/panda_reacher.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 import urdfenvs.panda_reacher
 import numpy as np
 
-goal = False
-obstacles = False
+goal = True
```
Comment (maxspahn) [inline examples/panda_reacher.py:5]: Could you set the defaults to False?

File: urdfenvs/sensors/obstacleSensor.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import numpy as np
```
Comment (maxspahn) [inline urdfenvs/sensors/obstacleSensor.py:1]: This file should not be needed as it is a copy of the file `urdfenvs/sensors/obstacle_sensor.py`
The naming was changed in #61.

**Review by maxspahn — CHANGES_REQUESTED** [review]
Sorry for this inconvenience. I should have noted that in #61. 
The file obstacleSensor should be removed however.

File: examples/obstacle_sensor.py (hunk 1 above)
Comment (GijsGroote) [reply examples/obstacle_sensor.py]: the import is now correct, and the ObstacleSensor file is no longer with us. obstacle_sensor is.

File: examples/panda_reacher.py (hunk 2 above)
Comment (GijsGroote) [reply examples/panda_reacher.py]: yes, done

**Review by maxspahn — APPROVED** [review]
Thanks for this quick fix.

**Pull request merged**